Server/datalayer/user.datalayer.js

```diff
@@ -8,6 +8,49 @@ class usersDatalayer {
     * Initializes the datalayer
     */
 
+    // Creates new user 
+    static createUser(reqbody) {
+	let user = new userModel(
+        {
+            username: reqbody.username,
+            email: reqbody.email,
+            password: reqbody.password,
+            firstname: reqbody.firstname,
+            lastname: reqbody.lastname
+        });
+
+        return user.save().catch(err => {
+        throw new error(`Not able to save - ${err}`);
```

Comment:
Correct the indentation of file, e.g. throw statement here

Reply:
Updated